### PR TITLE
Feature/3411 statistics

### PR DESCRIPTION
**Pull request by nickguendling**
## Description
Add statistics to the Home Screen. Also adds a Info Screen for the statistics feature.

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-3411

## Screenshots
IMG_0046
IMG_0048
IMG_0049
IMG_0050
IMG_0051
IMG_0052
IMG_0053
IMG_0054
IMG_0055
IMG_0058
IMG_0057
IMG_0056
IMG_0059
IMG_0060
IMG_0061
IMG_0062

### CODE REVIEW

**Review by tcfos — APPROVED** [review]

**Review by janetback — APPROVED** [review]

**Comment by Ein-Tim** [comment]
Thanks for all these screenshots! ❤

File: src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/HomeStatisticsCardViewModel.swift [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+		let dateFormatter = DateFormatter()
+		dateFormatter.dateStyle = .medium
+		dateFormatter.timeStyle = .none
+
+		return String(format: otherDateString, dateFormatter.string(from: self))
+	}
+
+}
+
+private extension SAP_Internal_Stats_KeyFigure {
```
Comment (marcussc) [inline src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/HomeStatisticsCardViewModel.swift:183]: Nitpicking: move this to its one file.

File: src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift [hunk 2]
```diff
@@ -974,6 +975,93 @@ enum AppStrings {
...
 		}
 	}
+
+	enum Statistics {
+
+		static let error = NSLocalizedString("Statistics_LoadingError", comment: "")
+
+		enum Card {
+
+			enum Infections {
```
Comment (marcussc) [inline src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift:985]: Love the nesting :)

File: src/xcode/ENA/ENA/Source/Views/BaseElements/ENALabel.swift [hunk 3]
```diff
@@ -7,6 +7,36 @@ import UIKit
 
 @IBDesignable
 class ENALabel: DynamicTypeLabel {
+
+	// MARK: - Init
+
```
Comment (marcussc) [inline src/xcode/ENA/ENA/Source/Views/BaseElements/ENALabel.swift:12]: Nice clean up :)

**Review by marcussc — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/HomeStatisticsCardViewModel.swift (hunk 1 above)
Comment (nickguendling) [reply src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/HomeStatisticsCardViewModel.swift]: Done 👍

**Pull request merged**